.travis.yml

```diff
@@ -4,6 +4,13 @@ jdk: oraclejdk8
 
 env:
   global:
+    - SONAR_HOST_URL="https://sonarcloud.io"
+
+    # SONAR_LOGIN value
+    # please note this value is available for pull requests from the same repository only
+    # see https://docs.travis-ci.com/user/pull-requests/#Pull-Requests-and-Security-Restrictions
```

Comment:
Ok I got it.
My pull request is from fork to main repository
That is why travis-ci is hiding secret values
In fact sonar is useless at pull requests until there is github commenting plugin is available at sonarcloud.io.
See my next comment